[quant] Adding max_pool1d deps

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#45175 [quant] Adding max_pool1d deps**
* #45152 [quant] torch.max_pool1d

Differential Revision: [D23856566](https://our.internmc.facebook.com/intern/diff/D23856566)

Commit:
[quant] Adding max_pool1d deps

[ghstack-poisoned]

ljk53 commented:
@z-a-f How did you create this PR? Did you add these deps manually or programmatically?
We usually update this file in a batch periodically (so you don't have to do it manually unless it's breaking some tests).

z-a-f commented:
> @z-a-f How did you create this PR? Did you add these deps manually or programmatically?
> We usually update this file in a batch periodically (so you don't have to do it manually unless it's breaking some tests).

Ah OK, I added it manually -- I thought I had to. Ignore this diff :)

Commit:
Update on "[quant] Adding max_pool1d deps"


Differential Revision: [D23856566](https://our.internmc.facebook.com/intern/diff/D23856566)

[ghstack-poisoned]

Commit:
Update on "[quant] Adding max_pool1d deps"


Differential Revision: [D23856566](https://our.internmc.facebook.com/intern/diff/D23856566)

[ghstack-poisoned]

Commit:
Update on "[quant] Adding max_pool1d deps"


Differential Revision: [D23856566](https://our.internmc.facebook.com/intern/diff/D23856566)

[ghstack-poisoned]

z-a-f commented:
Unnecessary PR
